examples/local-kind-environment/scripts/kind-with-registry.sh

```diff
@@ -16,10 +16,26 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 containerdConfigPatches:
 - |-
-  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:${reg_port}"]
+  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."${reg_name}:${reg_port}"]
     endpoint = ["http://${reg_name}:5000"]
 EOF
 
+# 3. Add the registry config to the nodes
```

Comment:
nit: Remove `3. ` (or add numbers to the remaining steps).

Reply:
Done.